### ISSUE TITLE
Can't set attributes to iframe

### ISSUE THREAD

**Issue by mykeysr** [officert/vue-friendly-iframe#9]
Tried this code,
```
 <div v-for="(floorPlan,index) in floorPlans">
                        <vue-friendly-iframe class="floor-plan-iframe"
                                             frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen :src="floorPlan.iframe_url" :ref="'iframeContent_'+index">
                            Your browser doesn't support iframes.
                        </vue-friendly-iframe>
                    </div>
```
But the out put is like,

```
<div data-v-1a3ed9ee="" class="vue-friendly-iframe floor-plan-iframe" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen="allowfullscreen">
<iframe crossorigin="anonymous" target="_parent" style="visibility: visible;">
</iframe>
</div>
```

The attributes `class="vue-friendly-iframe floor-plan-iframe" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen="allowfullscreen"` were should have been with the `<iframe>` tag instead of the parent `<div>`

**Comment by gr1zix** [comment]
same problem

**Comment by officert** [comment]
Fixed in 0.13.0 :

https://github.com/officert/vue-friendly-iframe/releases/tag/0.13.0

The `class` attribute isn't supported because Vue doesn't allow you to use it as a component prop name. So instead use `class-name`.
